main.tf

```diff
@@ -125,6 +126,24 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
     }
   }
 
+  origin {
+    origin_id = local.resize_function_origin_id
+    domain_name = "https://cesko.digital"
+    origin_path = '/api/resize'
```

Comment:
Tady by melo byt jenom `/api`, protoze chceme volat `https://data.cesko.digital/resize?url...`, takhle by tam to resize bylo dvakrat.

Mergnu to a pak kdyztak oddebuguju, at to nemusim spoustet lokalne.